release/s/sgaw_karen/source/help/sgaw_karen.php

```diff
@@ -0,0 +1,12 @@
+﻿<?php 
+  $pagename = '>ကညီကျိာ် S'gaw Karen Keyboard Help';

```

Comment:
Ah, the deployment is failing because of the single quote within the quotes.
Maybe should be
```suggestion
  $pagename = '>ကညီကျိာ် S\'gaw Karen Keyboard Help';
```

Reply:
Or use double quotes?